app/src/main/java/org/simple/clinic/summary/updatephone/UpdatePhoneNumberDialog.kt

```diff
@@ -139,7 +134,7 @@ class UpdatePhoneNumberDialog : AppCompatDialogFragment(), UpdatePhoneNumberDial
   override fun onStart() {
     super.onStart()
     onStarts.onNext(Any())
-    numberEditText.showKeyboard()
+    dialog?.numberEditText?.showKeyboard()
```

Comment:
Is adding null safety here the correct thing to do? We expect the dialog to be available, correct? In which case we should be using `!!` so we are notified of issues correctly.